addon/utils/wait-for.js

```diff
@@ -29,14 +30,14 @@ function runTimeout(func, wait) {
  * @method waitFor
  * @param  Function conditional
  * @param  Number wait
- * @return Ember.RSVP.Promise
+ * @return Promise
  * @public
  */
 export default function waitFor(conditional, wait = defaultWait, timeout = defaultTimeout) {
```

Comment:
I think we can remove this util with the equivalent [`waitUntil`](https://github.com/emberjs/ember-test-helpers/blob/master/addon-test-support/%40ember/test-helpers/wait-until.ts)

Reply:
deleted